backend/entities/office_hours/oh_ticket_entity.py

```diff
@@ -58,9 +62,75 @@ class OfficeHoursTicketEntity(EntityBase):
     )
 
     # One-to-one relationship of OfficeHoursTicket to UTA that has called it; optional field
-    caller_id: Mapped[int] = mapped_column(
+    caller_id: Mapped[int | None] = mapped_column(
         ForeignKey("academics__user_section.id"), nullable=True
     )
     caller: Mapped["SectionMemberEntity"] = relationship(
         back_populates="called_tickets"
     )
+
+    @classmethod
+    def from_model(cls, model: OfficeHoursTicket) -> Self:
+        """
+        Class method that converts an `OfficeHoursTicket` model into a `OfficeHoursTicketEntity`
+
+        Parameters:
+            - model (OfficeHoursTicket): Model to convert into an entity
+        Returns:
+            OfficeHoursTicketEntity: Entity created from model
+        """
+        return cls(
+            id=model.id,
+            caller_id=model.caller_id,
+            have_concerns=model.have_concerns,
+            caller_notes=model.caller_notes,
+            office_hours_event_id=model.office_hours_event_id,
```

Comment:
We have a naming difference here as well. I think on the entity side, our field is named `event_id` while on the model side, we have it name `office_hours_event_id`. Maybe we should should consider having them named the same?

But with that, I think this line would be `event_id=model.office_hours_event_id`

Only discovered this when I was trying to put test data into the tables!

Reply:
I changed the naming in both oh_ticket_entity.py and oh_ticket.py for it to be oh_event_id and oh_event; this helps avoid confusion with the separate event entity and is more standardized with the rest of our naming conventions.